styles/globals.css

```diff
@@ -32,6 +32,18 @@ body {
     'Liberation Mono', 'Courier New', monospace;
 }
 
+.dark .react-tabs__tab-panel {
+  background-color: #2222 !important;
+}
+
+.dark .iUjdA-d {
```

Comment:
These two styles are temporary as redoc don't have an option to change the background colors of these two classes.